doc/articles/uno-publishing-windows-packaged-signed.md

```diff
@@ -31,11 +31,28 @@ To package your app:
   msbuild /r /p:TargetFramework=net8.0-windows10.0.19041 /p:Configuration=Release /p:Platform=x64 /p:GenerateAppxPackageOnBuild=true /p:AppxBundle=Never /p:UapAppxPackageBuildMode=Sideloading /p:AppxPackageDir="C:/temp/output/" /p:AppxPackageSigningEnabled=true
   ```
 
+To package your app for the Microsoft App Store, the process is similar to creating a self-signed app package with just a minor difference:
+
+- Instead of linking to a self-signed certificate, associate your project with a Microsoft Store Application by right cliking on your project project in the solution explorer, then the **Publish**, **Associate App with Store...** menu item.
```

Comment:
```suggestion
- Instead of linking to a self-signed certificate, associate your project with a Microsoft Store Application by right-clicking on your project in the solution explorer, then the **Publish**, **Associate App with Store...** menu item.
```